third_party/Build.mk

```diff
@@ -70,5 +70,7 @@ third_party/localtests: cargo_version_check sandbox_setup
 
 .PHONY: build/cargo-host/release/elf2tab
 build/cargo-host/release/elf2tab: cargo_version_check sandbox_setup
-	cd third_party/elf2tab && \
+	cp -rp -t build third_party/elf2tab && \
```

Comment:
Yes.  Not likely to happen in a way that would break things, but indeed it could happen, so I've now added an rm -rf of build/ef2tab before it does a fresh copy.